src/App.js

```diff
@@ -7,9 +7,15 @@ import { Routes, Route } from "react-router-dom";
 import Reviews from "./components/Reviews";
 import IndividualReview from "./components/IndividualReview";
 import ReviewComments from "./components/ReviewComments";
+import Categories from "./components/Categories";
+import IndividualCategory from "./components/IndividualCategory";
+import { useState } from "react";
 
 function App() {
   const LoggedInUser = "weegembump";
+  const [commentCount, setCommentCount] = useState(0)
```

Comment:
Why are you keeping a commentCount in App state? Which review does this relate to and do you even use it? Remove this.